pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[project]
+name = "Rotary"
+version = "0.3.0"
```

Comment:
Change to 0.3.0-beta1 ?

Reply:
@jmtsuji I would leave this as is for now and just use numbers. I'm not sure if the pip library used is capable of parsing the  `-beta1` part or if it only likes numeric versions.

Reply:
@jmtsuji If we modify things it might break the installer which would not be good for a burst merging.

Reply:
@jmtsuji Addressed in https://github.com/rotary-genomics/rotary/commit/d47f9210b734a1c23a5a77cb0bf3029685905e3f